pywincffi/kernel32/handle.py

```diff
@@ -120,3 +120,50 @@ def WaitForSingleObject(hHandle, dwMilliseconds):
     error_check("WaitForSingleObject")
 
     return result
+
+
+def GetHandleInformation(hObject):
+    """
+    Returns properties of an object handle.
+
+    .. seealso::
+
+        https://msdn.microsoft.com/en-us/library/windows/desktop/ms724329(v=vs.85).aspx
+
+    :param handle hObject:
+        A handle to an object whose information is to be retrieved.
+    """
+    input_check("hObject", hObject, Enums.HANDLE)
+    ffi, library = dist.load()
+
+    lpdwFlags = ffi.new('LPDWORD')
+    code = library.GetHandleInformation(hObject, lpdwFlags)
+    error_check("GetHandleInformation", code=code, expected=Enums.NON_ZERO)
+
+    return lpdwFlags[0]
+
+
+def SetHandleInformation(hObject, dwMask, dwFlags):
+    """
+    Sets properties of an object handle.
+
+    .. seealso::
+
+        https://msdn.microsoft.com/en-us/library/windows/desktop/ms724935(v=vs.85).aspx
+
+    :param handle hObject:
+        A handle to an object whose information is to be set.
+
+    :param DWORD dwMask:
+        A mask that specifies the bit flags to be changed.
+
+    :param DWORD dwFlags:
+        Set of bit flags that specifies properties of the object handle.
+    """
+    input_check("hObject", hObject, Enums.HANDLE)
+    input_check("dwMask", dwMask, integer_types)
+    input_check("dwFlags", dwFlags, integer_types)
+    _, library = dist.load()
+
+    code = library.SetHandleInformation(hObject, dwMask, dwFlags)
```

Comment:
Instead of 

```python
code = library.SetHandleInformation(hObject, dwMask, dwFlags)
```

Could you do:
```python
code = library.SetHandleInformation(
    hObject, 
    ffi.cast("DWORD", dwMask), 
    ffi.cast("DWORD", dwFlags)
)
```

In #67, I added some reasoning behind this:
https://github.com/opalmer/pywincffi/commit/f6006d2aea78470e4715b7415f86aecdaa6742d1#diff-0f010e46bc45ff3689c2205c91cd62c3R232